modules/living_spaces_discussions/config/rewrite/group.role.project_space-f9845542f.yml

```diff
@@ -0,0 +1,5 @@
+permissions:
+  - 'create group_node:discussion_post entity'
+  - 'delete any group_node:discussion_post entity'
+  - 'update any group_node:discussion_post entity'
+  - 'view group_node:discussion_post entity'
```

Comment:
Project and Department spaces are installed in living_spaces_intranet. This module depends on living_spaces_discussions, so again rewrites won't apply and you will get and error.